test/utils/data.py

```diff
@@ -90,7 +91,9 @@ def get_prediction_job(cls, pid: Union[int, str]):
             # Change the typ column to forecast_type normally done after query
             out_dict["forecast_type"] = out_dict.pop("typ")
 
-        return PredictionJobDataClass(**out_dict)
+        return PredictionJobDataClass(**out_dict), ModelSpecificationDataClass(
```

Comment:
This function is still called get_prediction_job and it is used to get ModelSpecifications too. Either rename the function to `get_test_data` or `get_prediction_job_and_model_specs`. Or make a seperate function for `get_model_specs` to get test model specifications